common/app/targeting/CampaignAgent.scala

```diff
@@ -5,21 +5,28 @@ import com.gu.targeting.client.CampaignCache
 import conf.Configuration
 import scala.util.control.NonFatal
 import scala.concurrent.Future
+import conf.switches.Switches.Targeting
 
 object CampaignAgent extends Logging with ExecutionContexts {
-  private val agent = AkkaAgent[CampaignCache](CampaignCache(List()))
+  private val agent = AkkaAgent[CampaignCache](CampaignCache(Nil, None))
```

Comment:
Nil is the same as an empty list, I think it's considered more idiomatic. 

For the total count, we need a way to express a difference between there being 0 active campaigns and there being no data retrieved from the server. This could potentially be achieved with a boxed integer and using `null` but I think that would be bad practice.